Author: Thomas Renninger <trenn@suse.de>

Fix compiler warnings

oa_soap_sensor.c:2209:9: note: include ‘<string.h>’ or provide a declaration of ‘memcpy’

Index: openhpi-3.8.0/plugins/oa_soap/oa_soap_utils.h
===================================================================
--- openhpi-3.8.0.orig/plugins/oa_soap/oa_soap_utils.h	2018-10-04 16:19:50.682977324 +0200
+++ openhpi-3.8.0/plugins/oa_soap/oa_soap_utils.h	2018-10-04 16:20:29.393287079 +0200
@@ -41,6 +41,7 @@
 /* Include files */
 #include <math.h>
 #include <ctype.h>
+#include <string.h>
 
 #include "oa_soap_calls.h"
 #include "oa_soap.h"
